### PR TITLE
app: perf_overlay: disable CONFIG_ADSP_IDLE_CLOCK_GATING

### PR DESCRIPTION
**Pull request by abonislawski**
Perf build uses xtensa timer

### CODE REVIEW

File: app/perf_overlay.conf [hunk 1]
```diff
@@ -1,3 +1,4 @@
 CONFIG_PERFORMANCE_COUNTERS=y
 CONFIG_SYS_HEAP_RUNTIME_STATS=y
-CONFIG_TIMING_FUNCTIONS=y
\ No newline at end of file
+CONFIG_TIMING_FUNCTIONS=y
+CONFIG_ADSP_IDLE_CLOCK_GATING=n
```
Comment (kv2019i) [inline app/perf_overlay.conf:4]: @abonislawski I think a comment here would not hurt to explain use of xtensa timer and clock gating are not compatible. Even better would be to have this at Kconfig level but a comment here goes a long way.

**Review by kv2019i — APPROVED** [review]

**Review by lgirdwood — APPROVED** [review]
Yep, we should make sure Kconfig forbids perf with clock gating on Intel devices, but can be an incremental fix.

**Review by tmleman — APPROVED** [review]

**Comment by lgirdwood** [comment]
@abonislawski some CI tests have reported failures.

**Review by btian1 — APPROVED** [review]

**Comment by wszypelt** [comment]
@abonislawski @lgirdwood all green now in Internal Intel CI

**Pull request merged**